dynawo/sources/Models/Modelica/Dynawo/Examples/Nordic/BaseSystem/FullDynamicModel.mo

```diff
@@ -0,0 +1,562 @@
+within Dynawo.Examples.Nordic.BaseSystem;
```

Comment:
Same remarks as for other .mo models

Reply:
resolved, see [d146726](https://github.com/dynawo/dynawo/pull/2394/commits/d14672683da0d677556a70ce6fab3936ca672551)